src/main/java/ch/njol/skript/conditions/CondIsInvulnerable.java

```diff
@@ -18,27 +18,40 @@
  */
 package ch.njol.skript.conditions;
 
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import org.bukkit.GameEvent;
+import org.bukkit.GameMode;
 import org.bukkit.entity.Entity;
 
 import ch.njol.skript.conditions.base.PropertyCondition;
-import ch.njol.skript.doc.Description;
-import ch.njol.skript.doc.Examples;
-import ch.njol.skript.doc.Name;
-import ch.njol.skript.doc.Since;
 
 @Name("Is Invulnerable")
-@Description("Checks whether an entity is invulnerable.")
-@Examples("target entity is invulnerable")
-@Since("2.5")
-public class CondIsInvulnerable extends PropertyCondition<Entity> {
+@Description("Checks whether an entity or a gamemode is invulnerable.\nFor gamemodes, Paper and Minecraft 1.20.6 are required")
+@Examples({
+	"target entity is invulnerable",
+	"",
+	"loop {_gamemodes::*}:",
```

Comment:
```suggestion
	"loop all gamemodes:",
```
I believe this should be possible